hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -1455,7 +1462,7 @@ Stream<FileSlice> fetchAllFileSlices(String partitionPath) {
   /**
    * Default implementation for fetching latest base-files for the partition-path.
    */
-  public Stream<HoodieBaseFile> fetchLatestBaseFiles(final String partitionPath) {
+  Stream<HoodieBaseFile> fetchLatestBaseFiles(final String partitionPath) {
```

Comment:
Updating scope here since there is no read lock. The caller is not meant to use this directly.

Reply:
protected or private, people do not usually use the package scope.

Reply:
package private is actually very common. It is used to allow the other classes in the package to access this method while preventing others from calling it to give more control.